.mailmap

```diff
@@ -141,6 +141,7 @@ Dan Foreman-Mackey <foreman.mackey@gmail.com> Dan F-M <foreman.mackey@gmail.com>
 Johnny Greco <jgreco@astro.princeton.edu> Johnny <jgreco@astro.princeton.edu>
 Zé Vinicius <jvmirca@gmail.com> mirca <jvmirca@gmail.com>
 Michael Seifert <michaelseifert04@yahoo.de> --system <michaelseifert04@yahoo.de>
```

Comment:
Ah, seems like I tried to update my system gitconfig and messed up 😰

Reply:
water under the bridge, that was in the previous release 😅